src/pages/workspaces/workspace/analysis/file-utils.ts

```diff
@@ -47,6 +55,30 @@ export interface AnalysisFileStore {
   loading: boolean
```

Comment:
GitHub won't let me comment on the line, but it looks like `refresh` could be typed `() => Promise<void>`.

Reply:
Needed to do some `as [xxx]` syntax to get typescript happy with it, but `refresh` is now appropriately typed